src/main/java/com/google/cloud/discotoproto3converter/disco/Name.java

```diff
@@ -141,6 +141,23 @@ public String toLowerUnderscore() {
     return toUnderscore(CaseFormat.LOWER_UNDERSCORE);
   }
 
+  /**
+   * Returns the identifier in lower-underscore format but with the first uppercase letter if the
+   * original identifier started with an uppercase letter.
+   */
+  public String toUpperLowerUnderscore() {
```

Comment:
nit: suggest naming this `toCapitalizedLowerUnderscore`. Seems a bit more explanative.

Reply:
Yeah, it is a good name, much better than the one I had =) Changed.